geoportailv3/static/js/mymaps/mymapsdirective.js

```diff
@@ -38,6 +39,98 @@ app.module.directive('appMymaps', app.mymapsDirective);
  * @constructor
  * @export
  */
-app.MymapsDirectiveController = function() {};
+app.MymapsDirectiveController = function() {
+
+  /**
+   * @type {string}
+   * @private
+   */
+  this.defaultTitle_ = 'Map without title';
+
+  /**
+   * @type {boolean}
+   * @export
+   */
+  this.modifying = false;
+
+  /**
+   * @type {boolean}
+   * @export
+   */
+  this.choosing = false;
+
+  /**
+   * @type {Object?}
+   * @export
+   */
+  this.mymap = null;
+};
+
+
+/**
+ * Closes the current map.
+ * @export
+ */
+app.MymapsDirectiveController.prototype.close = function() {
+  // TODO ensure that modifications are saved.
+  this.mymap = null;
+};
+
+
+/**
+ * Creates a new map.
+ * @export
+ */
+app.MymapsDirectiveController.prototype.create = function() {
+  this.onChosen();
+};
+
+
+/**
+ * Open a map. Actually opens the map selector.
+ * @export
+ */
+app.MymapsDirectiveController.prototype.choose = function() {
+  this['choosing'] = true;
```

Comment:
Use the dot notation here, to be consistent with what you use in the constructor.